.travis.yml

```diff
@@ -7,7 +7,6 @@ env:
     # We want to run both (unit) tests and end to end tests as a matrix
     # https://docs.travis-ci.com/user/speeding-up-the-build/#Parallelizing-your-builds-across-virtual-machines
     - TEST_SUITE=test
-    - TEST_SUITE=test-e2e
```

Comment:
I believe I understand why you took this out, since the end to end tests would not run without an API key, but now this pull request does not have an end to end test.  Would it be okay to add this back in since the flipkart feature does not interfere with the e2e tests?